src/common/experienceData.json

```diff
@@ -0,0 +1,36 @@
+{
+    "data": [
+        {
+            "id": 1,
+            "name": "Magic EdTech",
+            "place": "Noida, India",
+            "logo": "https://ik.imagekit.io/zw5zu6q4kxa/magic_4d1UjcP2s.png",
+            "title": "Software Engineer",
+            "heading": "Sr. Associate (September 2020 - Present)",
+            "roles": [
+                "Working extensively in creating web based ed-gaming platforms in Reactjs.", "Leading the tech team", "Responsible for the agile development", "Working on building future proof e-learning games from kindergaten to 8th grade students.", "Extensively working with animations and overall state management."
```

Comment:
Can these different roles be in different lines?

That will increase coder readability

Reply:
done